packages/react/src/components/LoadingSpinner/index.tsx

```diff
@@ -10,68 +13,33 @@ export type LoadingSpinnerProps = {
 
 const LoadingSpinner = forwardRef<HTMLDivElement, LoadingSpinnerProps>(
   function LoadingSpinnerInner(
-    { size = 48, padding = 16, transparent = false, className },
+    { size = 48, padding = 16, transparent = false, ...props },
     ref
   ) {
+    const { progressBarProps } = useProgressBar(
+      useMemo(() => ({ isIndeterminate: true }), [])
+    )
```

Comment:
I think we don't need to use `useProgressBar` because the LoadingSpinner is always indeterminate and for simplification.

Reply:
ok. change it back to `role=“progressbar”`.